tests/test_arr_venue_v2.py

```diff
@@ -849,6 +851,32 @@ def test_june_cycle(self, client, openreview_client, helpers, test_client):
                 }
             )
         )
+
+        # Post duplicate and merge profiles
+        reviewer_two_merge_client.post_note_edit(
```

Comment:
the test is failing because the reviewer can not see the invitation, do you need to add it to the reviwers group?